tests/preprocessing/test_triplegs.py

```diff
@@ -162,6 +206,7 @@ def test_compare_to_old_trip_function(self):
         # generate trips and a joint staypoint/triplegs dataframe
         stps, tpls, trips = generate_trips(stps, tpls, gap_threshold=15)
         stps_, tpls_, trips_ = _generate_trips_old(stps, tpls, gap_threshold=15)
+        trips.drop(columns=["geom"], inplace=True)
```

Comment:
Here, we could also change the `_old_generate_trips` function to include the geometry, but it seems a bit overkill since the function is deprecated anyway. However, it would then of course be tested, whether iterating for each user leads also to the same geometry column as the function implemented now (at the moment I am sure that it yields the same result, since the test data was generated with my previous iterative version). Let me know if you think we should modify the `_old_generate_trips`.